CadRevealFbxProvider/BatchUtils/GeometryInstancer.cs

```diff
@@ -0,0 +1,39 @@
+namespace CadRevealFbxProvider.BatchUtils;
+using CadRevealComposer;
+using CadRevealComposer.Primitives;
+using CadRevealComposer.Tessellation;
+using System.Linq;
+using System.Security.Cryptography;
+
+public class GeometryInstancer
+{
+    public static void Invoke(List<CadRevealNode> nodes)
+    {
+        var instanceIDsAlreadyDone = new List<(ulong id, Mesh mesh)?>();
```

Comment:
This could be a dictionary i guess?

new List<(ulong id, Mesh mesh)?>
The value in the list should never be null so I would remove the nullable type